web-frontend/src/pages/Assets/AssetsList.tsx

```diff
@@ -17,8 +29,17 @@ export function AssetsList({ ...props }) {
           balance={token.balance}
           key={index}
           disable={token?.symbol === XMA ? true : false}
+          onDelete={() => {
+            handleDelete(token.assetAddress);
+          }}
         />
       ))}
+      {modalOpen && (
+        <DeleteAssetModal
+          tokenAddress={tokenAddress}
+          setModalOpen={setModalOpen}
```

Comment:
```suggestion
          onClose={() => setModalOpen(false)}
```

Reply:
onclose for me means : this is happening when we close... since here we now that we are closing why not ```ts close={()=>setModalOpen(false)}```

Reply:
yeah i agree with it !